index.html

```diff
@@ -211,14 +211,16 @@
   The web must enhance individuals' control and power
 </dfn></dt>
 <dd>
-We recognize that web technologies can be used by developers
-to manipulate people, complicate isolation and encourage addictive behaviors.
-We recognize these risks
-and seek to mitigate against them
-when creating these technologies and platforms.
-We will therefore favor a decentralized web architecture
+We recognize that web technologies can be used
+to manipulate and deceive people, complicate isolation, and encourage addictive behaviors
+(also known as <a href="https://en.wikipedia.org/wiki/Dark_pattern">dark patterns</a>).
+We seek to mitigate against these potential abuses
+when creating new technologies and platforms,
+and avoid introducing technologies that increase the chance of
+people being harmed in this way.
+As part of this, we favor a decentralized web architecture
```

Comment:
Would federated be what we are intending? Decentralized has a bit of a nuanced meaning depending on the audience (e.g. crypto) - maybe we should disambiguate the intent.

The web architecture itself is still decentralized, but the centralization of data and dependencies is the tricky bit.

Reply:
TIL... 

- https://networkcultures.org/unlikeus/resources/articles/what-is-a-federated-network/

> In politics, a federation is a union of states (or other entities) that are partially self-governing and independent but have transferred a set of responsibilities and duties to a central government that unites them. 

I think the web is federated not decentralized because:

-> https://www.iana.org/domains/root/servers

Reply:
```suggestion
As part of this, we favor a federated web architecture
```

Reply:
Opened a new issue on this because this was older language rather than part of this PR: https://github.com/w3ctag/ethical-web-principles/issues/54